app/user/templates/admin/deleteUser.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block title %}Delete user "{{ user.username | safe }}"{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <form class="form-deleteUser" method="POST">
+      {{ wtf.form_errors(form) }}
+      <h4 class="form-deleteUser-heading">Are you sure to delete user "{{ user.username | safe }}"?</h4>
```

Comment:
It would be cool if we could abstract this kind of "Are you sure you want to {{action}}" behaviour since we need that a lot